### PR TITLE
Upgrading django-cors-header.

### PR DESCRIPTION
**Pull request by awais786**
**PART1:**

https://openedx.atlassian.net/browse/BOM-1905
```
File "/edx/app/discovery/discovery/course_discovery/settings/test.py", line 1, in <module>
    from course_discovery.settings.base import *
  File "/edx/app/discovery/discovery/course_discovery/settings/base.py", line 7, in <module>
    from corsheaders.defaults import default_headers as corsheaders_default_headers
  File "/edx/app/discovery/discovery/.tox/django30/lib/python3.8/site-packages/corsheaders/__init__.py", line 1, in <module>
    from .checks import check_settings  # noqa: F401
  File "/edx/app/discovery/discovery/.tox/django30/lib/python3.8/site-packages/corsheaders/checks.py", line 7, in <module>
    from django.utils import six
ImportError: cannot import name 'six' from 'django.utils' (/edx/app/discovery/discovery/.tox/django30/lib/python3.8/site-packages/django/utils/__init__.py)
```

Change Log
https://github.com/adamchainz/django-cors-headers/blob/master/HISTORY.rst#300-2019-05-10

**NOTE:** Need to update the [configuration](https://github.com/edx/edx-internal/blob/master/ansible/vars/prod-edx.yml#L1156) as well

Configuration PR https://github.com/edx/edx-internal/pull/3434

----------------------------------------------------------------
**In 2nd part**
update the `django-autocomplete-light` as well.

### CODE REVIEW

**Comment by awais786** [comment]
**Part2** `django-autocomplete-light` [PR](https://github.com/edx/course-discovery/pull/2786) tests are running now with 6 failures on django3.0

**Review by mikix — COMMENTED** [review]
Looks like you got bit by the current bug around manual make-upgrades doing a different thing than our jenkins upgrade job. We have a ticket for that around, but haven't dug into it yet.

Can you just manually update `django-cors-headers` in the txt files? I don't see any changes in dependencies wrought by this upgrade, so we can just do the single package update.

File: requirements/constraints.txt [hunk 1]
```diff
@@ -7,9 +7,6 @@ elasticsearch<2
 # FIXME: 0.13+ requires python-slugify, which conflicts with unicode-slugify
 transifex-client<0.13
 
-# FIXME: 3+ require schemes in CORS_ORIGIN_WHITELIST URLs - remember to update configuration when you remove this
-django-cors-headers<3
-
 # jsonfield2 3.1.0 drops support for python 3.5
 jsonfield2<3.1.0
```
Comment (mikix) [inline requirements/constraints.txt]: This is not your current task, but seeing this here makes me realize we should make a swing for removing old-python constraints now that we're on 3.8.

**Review by mikix — APPROVED** [review]
Looks good! Once both edx-internal and configuration are settled, let's land this. Thanks for helping keep disco up to date.

File: requirements/constraints.txt [hunk 2]
```diff
@@ -13,7 +13,7 @@ elasticsearch-dsl>=7.2,<8.0
 transifex-client<0.13
 
 # FIXME: 3+ require schemes in CORS_ORIGIN_WHITELIST URLs - remember to update configuration when you remove this
-django-cors-headers<3
+django-cors-headers==3.2.0
```
Comment (waheedahmed) [inline requirements/constraints.txt:16]: Can't we remove this constraint now?

Comment (awais786) [reply requirements/constraints.txt:16]: We will go to 3.2.0 first, greater version has more settings related changes in 3.5.0
https://github.com/adamchainz/django-cors-headers/blob/main/HISTORY.rst

**Review by waheedahmed — APPROVED** [review]

**Pull request merged**